src/samples/server/Elsa.Samples.Server.Host/README.md

```diff
@@ -119,6 +119,17 @@ YesSql Postgre
     ...
 ```
 
+### Mandatory features
+
+"Elsa:Features:DefaultPersistence" is a mandatory feature resposible for Elsa Core persistence and must be enabled
```

Comment:
Actually, no persistence features should be mandatory to use Elsa. By default, it uses an in-memory provider. So I think we should update this comment to be optional.